creator-node/src/blacklistManager.js

```diff
@@ -29,13 +31,16 @@ class BlacklistManager {
     try {
       this.log('Initializing BlacklistManager...')
 
-      const { trackIdsToBlacklist, userIdsToBlacklist, segmentsToBlacklist } =
-        await this.getDataToBlacklist()
-      await this.fetchCIDsAndAddToRedis({
-        trackIdsToBlacklist,
-        userIdsToBlacklist,
-        segmentsToBlacklist
-      })
+      // Adding to redis only needs to be done once, but multiple workers might all run the app with their own BlacklistManager
+      if (cluster.worker?.id === 1) {
```

Comment:
is the reason worker 1 is doing all this stuff is because the master process doesn't init service registry?

Reply:
the only thought i have here is to not start an express server on the bull queue process
- use master process for bull queue processing
- don't start express server on worker 1 (but we have to make sure if the number of cores/workers spun up is at least 2)

Reply:
yeah

Reply:
sorry, didn't see your other comment - I migrated Bull to BullMQ and things seem to be working for multiple Bull processes now. the problem before with only having an express or bull server was that express needed to add jobs, so it needed access to bull queues

Reply:
if we can scale bull to multiple processes that's even better!

Reply:
right! it's been working on staging and Sid is reviewing now but if you have any concerns this is the implementation here in this PR